### ISSUE TITLE
Rename master to main

### ISSUE THREAD

**Issue by lawsie** [lawsie/guizero#502]
**Describe the solution you'd like**
Rename master branch to main

**Describe alternatives you've considered**
Considerations before doing this are [here](https://github.com/github/renaming)

**Comment by lawsie** [comment]
Looks like this is very easy to do. It'll require the branch for GitHub pages to publish to be changed, and a commit to the new 'main' to re-establish the GitHub pages site (i.e. the docs) so perhaps we could do it just before the next release @martinohanlon ?

**Comment by martinohanlon** [comment]
>perhaps we could do it just before the next release

Yep, good idea. It might worth a quick look through the docs to make sure nothing references master, I dont think it does though.

I would probably do this before adding CI.
